shacl12-test-suite/tests/sparql/property/property-eval-001.ttl

```diff
@@ -0,0 +1,57 @@
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix ex: <http://example.org/ns#> .
+@prefix mf: <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix sht: <http://www.w3.org/ns/shacl-test#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ex:Resource-uriLength
+	a sh:PropertyShape ;
+  sh:targetNode ex:Invalid ;  # The URI of this has 29 chars
+  sh:targetNode ex:Valid ;    # The URI of this has 27 chars
+	sh:name "URI length" ;

```

Comment:
Indents should only use spaces, else we'll have ongoing issues of differing indents because of differing tab settings and differing mixes of tabs and spaces. Here, I've changed two one-tab indents to eight-space indents, and two two-space indents to eight-space indents.
```suggestion
        a sh:PropertyShape ;
        sh:targetNode ex:Invalid ;  # The URI of this has 29 chars
        sh:targetNode ex:Valid ;    # The URI of this has 27 chars
        sh:name "URI length" ;

```